src/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Do List</title>
+
+  <!-- CSS only -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <!-- JavaScript Bundle with Popper -->
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous">
+  </script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <div class="main">
+    <div class="container">
+      <div class="heading">
+        <h2>Today's To Do</h2>
+        <i class="fa fa-refresh" aria-hidden="true"></i>
+      </div>
+      <hr>
+      <div><input type="text" placeholder="Add to your list..." id="add-task"><i class="fa-solid fa-arrow-turn-down-left"></i></div>
+      <hr>
+      <div id="show-tasks">
+
+
+      </div>
+    </div>
+    <div class="footer btn-secondary"><input type="button" value="Clear all completed" id="completed-task"></div>
```

Comment:
I would recommend we use `footer` element  make the page more semantic . Using `footer` also helps the page with assistive technology (e.g. screen readers)